go/store/nbs/store.go

```diff
@@ -1299,20 +1299,25 @@ func (nbs *NomsBlockStore) WriteTableFile(ctx context.Context, fileId string, nu
 		}()
 
 		return writeTo(f, rd, copyTableFileBufferSize)
-	}()
-
-	if err != nil {
-		return err
 	}
 
-	fileIdHash, ok := hash.MaybeParse(fileId)
+	return f()
```

Comment:
No reason to assign this func to a var if you're just going to immediately call it